pkg/server/admin_cluster_test.go

```diff
@@ -68,12 +68,6 @@ func TestAdminAPIDatabaseDetails(t *testing.T) {
 		}
 		assert.Equal(t, nodeIDs, resp.Stats.NodeIDs, "NodeIDs")
 
-		// TODO(todd): Find a way to produce a non-zero value here that doesn't require writing a million rows.
-		// Giving the test nodes on-disk stores (StoreSpec{inMemory: false, Path: "..."}) didn't work and was slow.
-		// Having a number would be nice here for completeness, but the real RangeCount above gives us enough
-		// confidence in the implementation in the meantime.
-		assert.Equal(t, uint64(0), resp.Stats.ApproximateDiskBytes, "ApproximateDiskBytes")
```

Comment:
Let's come back to first principles here: The "Approximate disk bytes" stats is a _very_ important UX outcome for the end-user. If that value is sometimes zero when it shouldn't, or sometimes non-zero when we want it to be zero, that's a real UX + observability bug. If the test is failing non-deterministically here, that's not a problem in the test: it's a problem in the logic, or our understanding of the feature.

There's two possible situations here (I don't know exactly which one is valid):

- the approximate disk bytes should be immediately non-zero after creating the test table above. This is the behavior I expect, especially after @zachlite 's recent changes.  If the test tries to observe zero, and we did the feature right, it's going to fail all the time. 
  In that case maybe the right change is `assert.NotEqual(t, 0, ApproxDiskBytes`)`.

- the approximate disk bytes is somehow computed lazily (we'd need to check with the KV team) so it can be zero for a while until it becomes non-zero.

  In that case the proper fix would be to use `SucceedsSoon` to check that it eventually becomes non-zero. Also in that case I would add some explanation (in a comment) about why the non-zero value lags, i.e. what mechanism in KV is responsible. That also probably deserves a clarifying release note so we properly document it.

Reply:
ack, I can ask the storage team about the latency, otherwise we'll need to be waiting > 1 minute according to Zach's linked comment. Thank you for the context!

Reply:
The question to the storage team could be "how can we ensure that a test that needs approx bytes can get it without waiting for a full minute" (i.e. make the test faster).